ethexe/runtime/src/wasm/interface/database.rs

```diff
@@ -27,6 +27,13 @@ interface::declare! {
     pub(super) fn ext_database_write_version_1(data: *const u8, len: i32) -> *const H256;
     pub(super) fn ext_get_block_height_version_1() -> i32;
     pub(super) fn ext_get_block_timestamp_version_1() -> i64;
+    pub(super) fn ext_gear_ri_update_state_hash_1(hash: *const H256);
+}
+
+pub fn update_state_hash(hash: &H256) {
```

Comment:
I'd consider moving it into separate ri mod (maybe context_ri) or something, since it doesnt access db actually

Reply:
atm seems enough to place todo for it

Reply:
Added TODO